frontend/packages/dev-console/src/components/import/__tests__/import-validation-utils.spec.ts

```diff
@@ -73,5 +73,17 @@ describe('ValidationUtils', () => {
         );
       });
     });
+
+    it('should throw an error if request is greater than limit', async () => {
```

Comment:
We might want a few additional tests here because this is so easy to get wrong. I'd suggest testing that you can enter either request or limit alone with no errors and another test to make a value like 5Mi is consider smaller than 1Gi.